gstorm/include/detail/ranges/vector.h

```diff
@@ -354,18 +339,26 @@ struct gvector : public traits::range_forward_traits<T>, public range::gvector_b
 private:
   // this is called by the gvector's base class gvector_base and is triggert after the
   // executor updates the gvectors cgh
-  virtual void updateAccessors() override {
-    auto acc = _buffer->template get_access<cl::sycl::access::mode::read_write>(*_cgh);
+  void updateAccessors() final override {
```

Comment:
Please have exactly one of `final` or `override`.

Reply:
final will be